core/trino-main/src/test/java/io/trino/sql/planner/TestLogicalPlanner.java

```diff
@@ -373,7 +374,7 @@ public void testDistinctOverConstants()
                                 ImmutableList.of("orderstatus"),
                                 "hash",
                                 anyTree(
-                                        project(ImmutableMap.of("hash", expression("combine_hash(bigint '0', coalesce(\"$operator$hash_code\"(orderstatus), 0))")),
+                                        project(ImmutableMap.of("hash", columnHashExpression("orderstatus")),
```

Comment:
Do we have some test which verifies the combined expression for multiple columns ?